src/build.d

```diff
@@ -210,7 +210,7 @@ Thus it first needs to be built and the executed.
 */
 auto opTabGen()
 {
-    auto opTabFiles = ["debtab.c", "optab.c", "cdxxx.c", "elxxx.d", "fltables.c", "tytab.c"];
+    auto opTabFiles = ["debtab.c", "optab.c", "cdxxx.c", "elxxx.d", "fltables.d", "tytab.c"];
```

Comment:
Added this commit to fix the CircleCi build failures (the `build.d` script is tested there.)

Reply:
ok